nimble.nimble

```diff
@@ -11,7 +11,7 @@ installExt = @["nim"]
 
 # Dependencies
 
-requires "nim >= 0.13.0"
+requires "nim >= 0.13.0", "gitapi"
```

Comment:
nimble shouldn't depend on external packages.

Reply:
ok, if the git init function is accepted, i can refactor the code later.